src/ucs/sys/event_set.h

```diff
@@ -0,0 +1,29 @@
+#ifndef UCS_EVENT_SET_H
+#define UCS_EVENT_SET_H
+
+#include <ucs/type/status.h>
+
+typedef struct ucs_sys_event_set {
```

Comment:
this struct should be defined in C file, to avoid including config.h in this file in the future